src/helpers/fixtures.js

```diff
@@ -1,8 +1,9 @@
 'use strict';
 
-let _ = require('lodash');
-let omitDeep = require('omit-deep');
-let xhrHelper = require('./xhr');
+const _ = require('lodash');
```

Comment:
Yes, I've done this in a separate 'refactor everything' branch so omitted it from this one.